tests/integration/hive_metastore/test_federation.py

```diff
@@ -0,0 +1,10 @@
+from databricks.labs.blueprint.commands import CommandExecutor
+from databricks.labs.ucx.hive_metastore.federation import DbfsRootResolver
+
+
+def test_dbfs_root_resolver(ws, env_or_skip):
+    command_executor = CommandExecutor(ws.clusters, ws.command_execution, lambda: env_or_skip("DATABRICKS_CLUSTER_ID"))
+    root_resolver = DbfsRootResolver(command_executor)
```

Comment:
add a unit test in the `tests/unit/hive_metastore` folder
```
command_executor_mock = create_autospec(CommandExecutor)
command_executor_mock.run.return_value = 's3://...'
```

example of a unit test:
https://github.com/databrickslabs/ucx/blob/main/tests/unit/hive_metastore/test_locations.py#L98-L114

Reply:
Sorry for the late response. I am not sure what the above unit test will actually be testing if the command executor itself a mock. I added a unit test as per your suggestion above. Let me know if you had something else in mind.